docs/guides/postinstall.md

```diff
@@ -172,9 +190,105 @@ S3 suspend has been broken since macOS Sonoma, it has never been fixed, but this
      [Install]
      WantedBy=sleep.target
      ```
+3. Check your modprobe location
+     ```bash
+     which modprobe
+     which rmmod
+     ```
+and fix the pathes in service script if they differ for your system
+
+4. If you having problems with touchbar being dead after restoring state from suspend then you might as well try the following version of the script:
+     ```service
+     [Unit]
+     Description=Disable and Re-Enable Apple BCE Module (and Wi-Fi)
+     Before=sleep.target
+     StopWhenUnneeded=yes
+     
+     [Service]
+     User=root
+     Type=oneshot
+     RemainAfterExit=yes
+     
+     ExecStartPre=+/usr/sbin/modprobe -r hid_appletb_kbd
+     ExecStart=+/usr/sbin/modprobe -r brcmfmac_wcc
+     ExecStart=+/usr/sbin/modprobe -r brcmfmac
+     ExecStart=+/usr/sbin/rmmod -f apple-bce
+     
+     ExecStop=+/usr/sbin/modprobe apple-bce
+     ExecStop=+/usr/sbin/modprobe brcmfmac
+     ExecStop=+/usr/sbin/modprobe brcmfmac_wcc
+     
+     
+     [Install]
+     WantedBy=sleep.target
+     ```
+
+This script was written specifically for Debian Bookworm tiny-dfr touchbar issue, but you will loose ability to control keyboard brightness with touchbar. For some reason unloading ```hid_appletb_kbd``` helps to maintain touchbar in alive state but then this module trips and doesnt work anymore.
+You can still control keyboard brightness with just bash tho:
+     
+     echo 8192 | sudo tee /sys/class/leds/\:white\:kbd_backlight/brightness
+
+The path shown in this example is relevant for only MacBookPro16,x and MacBookAir9,1
+     
+Also you can add a udev rule to give a group write permissions for the brightness value. Add to a new `leds.rules` file in `/etc/udev/rules.d`:
+
+     ACTION=="add", SUBSYSTEM=="leds", RUN+="/bin/chgrp GROUP_NAME $sys$devpath/brightness", RUN+="/bin/chmod g+w $sys$devpath/brightness"
 
-3. Enable the service by running: `sudo systemctl enable --now suspend-fix-t2.service`
+5. Enable the service by running: `sudo systemctl enable --now suspend-fix-t2.service`
 
 !!! note
-    This seems to be working only on Arch with `CONFIG_MODULE_FORCE_UNLOAD=y` in the kernel config.
-    To check, run: `zcat /proc/config.gz | grep "CONFIG_MODULE_FORCE_UNLOAD"`
+    This seems to be working with `CONFIG_MODULE_FORCE_UNLOAD=y` in the kernel config when kernel is compiled with [kernel compilation instructions](https://wiki.t2linux.org/guides/kernel/).
+    
+To check, run: 
+`zcat /proc/config.gz | grep "CONFIG_MODULE_FORCE_UNLOAD"` on arch
+or
+`cat /boot/config-$(uname -r) | grep "CONFIG_MODULE_FORCE_UNLOAD"` on Debian-based distros.
+
+Without this config option you wont be able to unload the required modules, they will be busy.
+
+6. If touchbar occasionally does not work on boot but works after suspend+restore then you can place this workaround somewhere late after boot
```

Comment:
Won't it be better to just keep this and remove the extra suspend.service you made here https://github.com/t2linux/wiki/pull/556/files#diff-a9bb6c3fdd453b5d31bf4ce51c2122abccd3d3f1d21d0c2c78f5e783a7bc1b87R212

Reply:
Again, this hasn't been addressed yet